.golangci.yml

```diff
@@ -17,7 +17,7 @@ linters-settings:
     local-prefixes: github.com/SKF/go-rest-utility
   gocyclo:
     # minimal code complexity to report, 30 by default (but we recommend 10-20)
-    min-complexity: 10
+    min-complexity: 11
```

Comment:
haha :)

Reply:
It complained about the CredentialsTokenProvider.signIn method. Guessing it is too many branches. :smile: